frontend/src/sass/layout/_footer.scss

```diff
@@ -4,9 +4,76 @@
 
 .footer {
     background-color: $colr-pnl-footer;
-    height: 5em;
-    margin: 0;
     width: 100%;
     color: $colr-txt;
-    text-align: center; 
+    text-align: center;
+    padding: 2em 0;
+    text-transform: uppercase;
+    font-size: .7em;
+    color: $colr-footer;
+
+    a {
+        color: $colr-footer;
+        margin-right: 15px;
+
+    }
+
+    ul {
+        padding: 0;
+        margin: 0;
+
+        li {
+            display: inline-block;
+
+        }
+    }
+
+    &__navigation {
+        display: flex;
```

Comment:
Flexbox has a very bad support on IE, I'll consider dropping it. Since is used elsewhere in the project is fine to have it here.